src/main/kotlin/racingcar/RacingCarController.kt

```diff
@@ -0,0 +1,58 @@
+package racingcar
+
+import camp.nextstep.edu.missionutils.Console
+
+class RacingCarController(private val view: RacingCarView, private val model: RacingCarModel) {
+    fun race() {
+        signUpRacer() //자동차 경주 입력 요청 및 처리
+        enterPlayTime() //이동 시도 횟수 입력 요청 및 처리
+        initializationMoveDistance() //이동 거리 리스트 초기화
+        requestPrintResultText() //실행 결과 텍스트 사전 출력
+        for (sequence in FIRST_SEQUENCE until model.racerCrew.playTime) { //이동 시도 횟수 반복
+            judgeRace() //정지 혹은 전진 판단 및 반영
+            printRace() //정지 혹은 전진 판단 결과 출력
+        }
+        judgeChampion() //최종 우승자 도출
+        printChampion() //최종 우승자 출력
+    }
+
+    private fun signUpRacer() {
+        view.printRequestRacer()
+        val unverifiedRacerName = RacerName(Console.readLine())
+        val verifiedRacer: MutableList<String> = unverifiedRacerName.verifyRacerName()
+        model.requestUpdateRacerName(verifiedRacer)
+    }
+
+    private fun enterPlayTime() {
+        view.printRequestPlayTime()
+        val unverifiedPlayTime = RaceTime(Console.readLine().toInt())
```

Comment:
요구사항: 사용자가 잘못된 값을 입력할 경우 IllegalArgumentException을 발생시킨 후 애플리케이션은 종료되어야 한다.
해당 부분에서 정수가 아닌 문자열이 들어온 경우 IllegalArgumentException 가 발생하도록 하면 좋을 것 같습니다.
```suggestion
       val raceTime = Console.readLine().toIntOrNull() ?: throw IllegalArgumentException
       val unverifiedPlayTime = RaceTime(raceTime)
```

Reply:
아 단순하게 정수로만 입력 받고자하는 의도로 작성 했던 코드였는데 제 고민이 제시해주신 코드까지 도달하지 못한 거 같아요ㅠㅠ 감사합니다